azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/service/process/ProcessFieldResponseModel.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * azure-boards-common
+ *
+ * Copyright (c) 2020 Synopsys, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.azure.boards.common.service.process;
+
+public class ProcessFieldResponseModel {
+    private Boolean allowGroups;
+    private String customization;
+    private Object defaultValue;
```

Comment:
Is this used at all?  Does this need to be a generic object type or is there a specific object representation for the default value?

Reply:
According to the docs, it is an `Object`, but I should probably change it to `JsonObject` just in case we ever need to drill down into its contents.